aws-cs-fargate/App/Program.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace dotnet_core_tutorial
```

Comment:
Rename the namespace to something .NET-y

Reply:
Will do, all of this is whatever `dotnet new` spit out, fwiw.